src/main/java/seedu/address/logic/parser/FindEventsCommandParser.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.util.Arrays;
+
+import seedu.address.logic.commands.FindEventsCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.event.EventTitleContainsKeywordsPredicate;
+
+/**
+ * Parses input arguments and creates a new FindEventsCommand object
+ */
+public class FindEventsCommandParser implements Parser<FindEventsCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the FindEventsCommand
+     * and returns a FindEventsCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
```

Comment:
Just a missing fullstop here
```suggestion
     * @throws ParseException if the user input does not conform the expected format.
```